composer.json

```diff
@@ -5,12 +5,12 @@
     "homepage": "http://github.com/friendsofcake/CakePdf",
     "license": "MIT",
     "require": {
-        "php": ">=5.4.16",
-        "cakephp/cakephp": "~3.1"
+        "cakephp/cakephp": "^3.4"
     },
     "require-dev": {
-        "phpunit/phpunit": "<6.0",
-        "dompdf/dompdf":"~0.7"
+        "phpunit/phpunit": "^5.7|^6.0",
+        "dompdf/dompdf":"~0.7",
+        "mpdf/mpdf": "^7.0"
```

Comment:
Is it on purpose to include both mpdf and dompdf for all users of CakePdf? 🤔

Reply:
It's under `require-dev` for the test suite.